phpBB/language/en/cli.php

```diff
@@ -75,6 +75,7 @@
 	'CLI_DESCRIPTION_REPARSER_REPARSE'					=> 'Reparses stored text with the current text_formatter services.',
 	'CLI_DESCRIPTION_REPARSER_REPARSE_ARG_1'			=> 'Type of text to reparse. Leave blank to reparse everything.',
 	'CLI_DESCRIPTION_REPARSER_REPARSE_OPT_DRY_RUN'		=> 'Do not save any changes; just print what would happen',
+	'CLI_DESCRIPTION_REPARSER_REPARSE_OPT_FORCE_BBCODE'	=> 'Enforce reparsing BBCode in posts unconditionally. Please note that it will parse all BBCode in all posts, even if those were disabled or disallowed during posting.',
```

Comment:
Force reparsing BBCodes without exception. Be aware that this will apply to all BBCodes in all posts, including any that were disabled or restricted at the time of original posting.

Reply:
`Force reparsing BBCodes without exception. Be aware that this will apply to all BBCodes, including any that were disabled or restricted at the time of original text creation.` <- will this be good?
The point is that BBCodes are used not in posts only but also in many other places (signatures, poll titles, PMs, group descriptions, forum rules, forum descriptions, contact admin info). So it should be more common and not restricted only to posts/posting.

Reply:
Yes

Reply:
Hm, not sure if this is clear enough. It doesn't just apply to all BBCodes but rather it enables all BBCodes and related settings.

Reply:
This then?
_Force re-parsing BBCodes without exception. Be aware that this will enable all BBCodes and related settings, including any that were disabled or restricted at the time of original message processing._

Reply:
This probably becomes less clear the more words are involved. Actually this option will just enable all BBCodes even those were disabled in the text to be processed, that's it.
The option works **on per-item basis** (a message, a signature, a poll etc) and doesn't switch any related settings.

Reply:
Something like
```
Force re-parsing BBCodes without exception. Be aware that this will override any settings which were disabling or restricting the use of BBCode at the time of the text creation and will enable all BBCodes in the text of every item to be re-parsed.
```

Reply:
Thaty's even more confusing and going in circles, LOL

Maybe:
_Re-parse all BBCodes without exception. This action will reprocess and render all BBCodes, including those that were previously disabled, meaning any disabled BBCodes from the original parsing will be enabled and fully rendered._

Reply:
May be even shorter?

_Re-parse all BBCodes without exception. Precaution: BBCodes that were disabled will be enabled, reprocessed and fully rendered._

Reply:
_Re-parse all BBCodes without exception. Note that any previously disabled BBCodes will be reprocessed, enabled, and fully rendered._